browse/static/js/coreRecommender.js

```diff
@@ -0,0 +1,62 @@
+function getCORERecCookie() {
+    var coreCookie = "arxiv_core";
+    var dc = document.cookie;
+    var prefix = coreCookie + "=";
+    var begin = dc.indexOf(prefix);
+    var end = document.cookie.indexOf(";", begin);
+    var returnVal;
+    if (end == -1) {
+        end = dc.length
+    }
+    returnVal = decodeURIComponent(dc.substring(begin + prefix.length, end));
+    return returnVal
+}
+function setCORERecCookie() {
+    var coreCookie = "arxiv_core";
+    var value = "disabled";
+    var cv = getCORERecCookie();
+    console.log("cv");
+    console.log(cv);
+    if (cv == "disabled") {
+        value = "enabled"
+    } else if (cv == "enabled") {
+        value = "disabled"
+    }
+    var coreCookieValue = value;
+    var today = new Date;
```

Comment:
I would use `new Date()` but it's minor.

However, syntax without braces is a modern one, so if you use it there is no need to use `var` and concatenate strings.